.prettierrc.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = {
+  overrides: [
+    {
+      files: '*.{js,ts}',
+      options: {
+        singleQuote: true,
```

Comment:
would be in favor of addiing `"semi": false` (for all our DazzlingFugu projects), maybe we can sync about this on discord

Reply:
See PR's description.

Reply:
J'ai revert et fais avec le `semi: false` car je sens que personne ne va vouloir suivre sinon...
Mais je maintiens que l'on devrait matcher la configuration par défaut de projets Ember.js, donc avec `semi: true`, ça nous fera 1000 fois moins chier qu'être le seul petit groupe qui fait avec `semi: false`.

Reply:
Aucun soucis de mon côté si on laisse la configuration par défaut, je veux juste pas qu'on se retrouve avec des personnes qui mettent pas le `;` et d'autres qui le mettent 

Si on a un hook ou un job (je sais pas si le `yarn lint:js` le détecte?) qui vérifie ça je pense qu'on peut garder `semi: true`, dans le fond nos repos sont pas très actifs non plus (à part tes contributions ❤️ ) donc ça impactera pas grand monde

Reply:
Ouais il y a des trucs pour automatiser la détection, (1) perso je l'ai déjà en live dans mon éditeur donc pas de plus-value pour ma part et (2) comme tu dis "nos repos sont pas très actifs" donc a-t-on vraiiiiiment besoin de ça, je me pose la question et (3) la CI s'exécute en 3 minutes donc c'est pas non plus couteux de push/au mince erreur/je corrige/je repush, genre ça va c'est largement acceptable.

Reply:
J'en profite, même si rien de pressé du tout, j'ai fais la "même PR" pour `ember-feature-controls`:
https://github.com/DazzlingFugu/ember-feature-controls/pull/255